src/routes/cart.route.js

```diff
@@ -0,0 +1,13 @@
+import { Router } from 'express';
+import cookieParser from 'cookie-parser';
+import { isAuthenticated } from '../middleware';
+import { cartController } from '../controllers';
+import { asyncWrapper } from '../helpers';
+
+const router = Router();
+
+router.use(cookieParser());
```

Comment:
do u need this cookie parser?

Reply:
Since I had previously used it, I had forgotten to take it out. but is now deleted